src/scribe_data/extract_transform/wikidata/update_data.py

```diff
@@ -130,108 +136,46 @@
     # First format the lines into a multi-line string and then pass this to SPARQLWrapper.
     with open(query_path, encoding="utf-8") as file:
         query_lines = file.readlines()
-    sparql.setQuery("".join(query_lines))
-
-    results = None
-    try:
-        results = sparql.query().convert()
-    except HTTPError as err:
-        print(f"HTTPError with {query_path}: {err}")
-
-    if results is None:
-        print(f"Nothing returned by the WDQS server for {query_path}")
+    
+    
+    if any("OFFSET" in line for line in query_lines):
+        
+        language_id = extract_language_id(query_lines)
+        Total_nouns = get_total_nouns(language_id)
+
+        batch_size = 10000  #adjust as needed {{Same as limit}}
+     
+        # Calculate the number of iterations needed
```

Comment:
Minor nitpick on this, if the comment is it's own line, then it's Python standard to include a period :) I can add these, but for later 🚀

Reply:
alright